conda_build/skeletons/cran.py

```diff
@@ -1284,6 +1321,14 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
                                 package_dicts.update({lower_name: {'inputs': inputs_dict}})
                                 package_list.append(lower_name)
 
+            for name in dep_dict:
+               # print('seek {} in for_cran'.format(name))
```

Comment:
Could you please delete these commented codes?

Reply:
Removed the commented code lines